students/grzegorz_bajorski/lesson_02_flow_control/the_number_of_elements_equal_to_the_maximum.py

```diff
@@ -0,0 +1,13 @@
+maximum = 1
+count = 0
+x = 1
+
+while x != 0:
+    x = int(input())
+    if x > maximum:
+        maximum = x
+        count = 1
+    elif x == maximum:
+        count += 1
+print(count)
+
```

Comment:
Too many empty lines at the end of the file. According to PEP8 should be only one.